scripts/add_translation.py

```diff
@@ -71,18 +80,47 @@ def get_credentials():
     return credentials
 
 
+def display_languages(service):
+    print 'List of available languages:\n----------------------------'
+    data = service.files().list(q="'" + TRANSLATION_FOLDER_ID + "'" + " in parents").execute()
+    for f in data['files']:
+        if f['mimeType'].endswith('.folder'):
+            LANGUAGE_LIST[f['name']] = f['id']
+
+    # sort the languages alphabetically
+    sorted_list = [e for e in sorted(LANGUAGE_LIST)]
+    id = 1
+    # store the generated IDs to match against user input.
+    language_id = {}
+    for i in sorted_list:
```

Comment:
For future reference, you can get a counter using `for i, item in enumerate(sorted_list)`, and eliminate the `id` variable and manual incrementing altogether.